### ISSUE TITLE
[ARM64] Switch to official redis, mariadb and rebbitmq images

### ISSUE THREAD

**Issue by ihor-sviziev** [davidalger/warden#485]
Currently, warden is using Redis images w/o any modifications.
The only diff is the official images do support ARM architecture, while the warden's images don't.

Just replacing redis to the official image using creating a .warden/warden-env.darwin.yml file removes the x86_64 emulation on ARM machines.
```
version: "3.5"
services:
  redis:
    image: redis:5.0
```

Also @drpayyne did the same in https://github.com/drpayyne/warden-multi-arch/commit/a4d7e001c08de174667a1326fcd0fa6a192e70c3#diff-d22b3969c3251dfa2e3fa7273d0aea01b9cc59b598d066f8f613ea2968ca3f2aR5

Basically, the same situation with 
- mariadb: https://github.com/drpayyne/warden-multi-arch/commit/a4d7e001c08de174667a1326fcd0fa6a192e70c3#diff-4a2741d6c01bfe2dc20298c80e69d6b56c4a0966b57527a57bb33f1d78ee966bR9
- rabbitmq: https://github.com/drpayyne/warden-multi-arch/commit/a4d7e001c08de174667a1326fcd0fa6a192e70c3#diff-4845bf27c9c80c4e34db649440c834232240ed1053dd7a7ea55f1e21953d545dR5

**Comment by ihor-sviziev** [comment]
@davidalger, what do you think about it?
Also an option create multi-arch builds for arm

**Comment by davidalger** [comment]
See #505, have added multi-arch builds for these. Although atm these have no modifications, it's simpler to have them all mirrored (despite having been a bit more painful with arm64 support not transparently being added in).
